### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #43 

Some updates and reformatting for the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,97 @@
-# Software Stack
 
 ![Ubuntu](https://github.com/noaa-emc/hpc-stack/workflows/Build%20Ubuntu/badge.svg)
 ![macOS](https://github.com/noaa-emc/hpc-stack/workflows/Build%20macOS/badge.svg)
 
-This repository provides a unified, shell script based build system for building commonly used software stack.
+# hpc-stack
+
+This repository provides a unified, shell script based build system
+for building the UFS weather model software stack. This is run on NOAA
```
Comment (aerorahul) [inline README.md]: This software stack builds the software stack for more than just the UFS weather model (and hence the name hpc-stack and not ufs-stack).

File: README.md [hunk 2]
```diff
@@ -1,49 +1,97 @@
...
+HPC systems to consistently and reliably build and install all the
+libraries and tools needed for the NOAA Universal Forecast System
```
Comment (aerorahul) [inline README.md]: Again, the software stack is broader than the UFS.

File: README.md [hunk 3]
```diff
@@ -1,49 +1,97 @@
...
-
-_**NOTE: To enable a boolean flag use a single-digit `Y` or `T`.  To disable, use `N` or `F` (case insensitive)**_
-
-- **PKGDIR:** is the directory where tarred or zipped software files will be downloaded and compiled.  Unlike `PREFIX`, this is a relative path, based on the root path of the repository.  Individual software packages can be downloaded manually to this directory and untarred, but this is not required.  Build scripts will look for directory `pkg/pkgName-pkgVersion` e.g. `pkg/hdf5-1_10_3`.
-
-- **LOGDIR:** is the directory where log files from the build will be written, relative to the root path of the repository.
-
-- **OVERWRITE:** If set, this flag will cause the build script to remove the current installation, if any exists, and replace it with the new version of each software package in question.  If this is not set, the build will bypass software packages that are already installed.
+The first step is to chose the **COMPILER** and the **MPI** specify
```
Comment (aerorahul) [inline README.md]: `choose` not `chose`

**Review by aerorahul — CHANGES_REQUESTED** [review]
I just have one comment that this stack is not just for UFS.  It is broader than that and either we leave it that way or spell out (atleast) the major applications that use this stack.

The formatting looks good.  Thanks!

File: README.md (hunk 1 above)
Comment (edwardhartnett) [reply README.md]: Awesome! What should I add? JEDI? What else?

File: README.md (hunk 3 above)
Comment (edwardhartnett) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (aerorahul) [reply README.md]: how about this ...

... for building the software stack used in NOAA applications such as the UFS model, JEDI-based data assimilation apps, etc.

File: README.md [hunk 4]
```diff
@@ -1,49 +1,97 @@
...
 ![macOS](https://github.com/noaa-emc/hpc-stack/workflows/Build%20macOS/badge.svg)
 
-This repository provides a unified, shell script based build system for building commonly used software stack.
+# hpc-stack
+
+This repository provides a unified, shell script based build system
+for building the UFS weather model software stack. This is run on NOAA
+HPC systems to consistently and reliably build and install all the
+libraries and tools needed for the NOAA Universal Forecast System
+(UFS) and releated products.
```
Comment (aerorahul) [inline README.md]: `releated` -> `related`

File: README.md [hunk 5]
```diff
@@ -1,49 +1,97 @@
...
+
+This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS)
```
Comment (aerorahul) [inline README.md:13]: Is it?

File: README.md (hunk 1 above)
Comment (edwardhartnett) [reply README.md]: Does JEDI have a website?

Not keen on "etc." What else other than UFS and JEDI, if anything?

File: README.md (hunk 4 above)
Comment (edwardhartnett) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (aerorahul) [reply README.md]: UFS and JEDI are fine.
I didn't see a UFS website other than the github page.
JEDI RTD is at: https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/

Comment (edwardhartnett) [reply README.md]: OK, I've edited - take a look at the new version.

File: README.md [hunk 6]
```diff
@@ -1,49 +1,99 @@
...
 
 ![Ubuntu](https://github.com/noaa-emc/hpc-stack/workflows/Build%20Ubuntu/badge.svg)
 ![macOS](https://github.com/noaa-emc/hpc-stack/workflows/Build%20macOS/badge.svg)
 
-This repository provides a unified, shell script based build system for building commonly used software stack.
+# hpc-stack
+
+This repository provides a unified, shell script based build system
+for building software stack needed for the NOAA [Universal Forecast System
+(UFS)](https://github.com/NOAA-GSD/ufs-weather-model) and related
```
Comment (aerorahul) [inline README.md]: Please use `https://github.com/ufs-community/ufs-weather-model`  The one being pointed to currently is a fork.

Comment (edwardhartnett) [reply README.md]: Done.

**Comment by edwardhartnett** [comment]
OK, all outstanding comments have been addressed. Unless there are new comments, this is ready to merge.

**Review by aerorahul — APPROVED** [review]
Thanks for correcting the links and formatting.

**Pull request merged**